### PR TITLE
Add extend about data method to gui utils

### PR DESCRIPTION
**Pull request by AIIX**
- Adds a method that skills, packages and plugins can call to add about data / version data for them in the about gui page

Supplements https://github.com/OpenVoiceOS/ovos-core/pull/185

### CODE REVIEW

File: ovos_utils/gui.py [hunk 1]
```diff
@@ -49,6 +49,15 @@ def can_use_gui(bus=None, local=False):
         return can_use_local_gui()
     return can_use_local_gui() or is_gui_connected(bus)
 
+def extend_about_data(about_data):
+    bus = get_mycroft_bus()
```
Comment (JarbasAl) [inline ovos_utils/gui.py:53]: usually this should be avoided when possible to not create a new websocket connection, can you add `bus=None`to the arguments and make this `bus = bus or get_mycroft_bus()`? that allows to re utilize a open websocket which will already exist in basically everything that will consume this new method

Comment (AIIX) [reply ovos_utils/gui.py]: done

File: ovos_utils/gui.py [hunk 2]
```diff
@@ -49,8 +49,8 @@ def can_use_gui(bus=None, local=False):
         return can_use_local_gui()
     return can_use_local_gui() or is_gui_connected(bus)
 
-def extend_about_data(about_data):
-    bus = get_mycroft_bus()
+def extend_about_data(bus=None, about_data):
```
Comment (JarbasAl) [inline ovos_utils/gui.py:52]: this needs to be reversed, `about_data` needs to come first since its optional but `bus` isnt

Comment (AIIX) [reply ovos_utils/gui.py]: done

**Review by JarbasAl — APPROVED** [review]

**Pull request merged**